src/components/list/ListForm.vue

```diff
@@ -152,23 +162,31 @@ export default {
 
       return filtersNames.length === new Set(filtersNames).size;
     },
-    deleteList(id) {
-      this._deleteList(id);
-      this.closeListForm();
-    },
-    submitList() {
+    saveList() {
+      this.isRequestProcessing = true;
       if (this.verifyFilters()) {
         if (this.edittingListObj) {
-          this._updateList(this.list);
+          this._updateList(this.list).then(() => {
+            this.isRequestProcessing = false;
+            this.closeListForm();
+          });
         } else {
-          this._addList(this.list);
+          this._addList(this.list).then(() => {
+            this.isRequestProcessing = false;
+            this.closeListForm();
+          });
         }
-
-        this.closeListForm();
       } else {
         this.errorMessage = 'same names in filters';
       }
```

Comment:
also here - if filters are not verified - status still will be true. I think it's better to put directly before request.